gratipay/utils/markdown.py

```diff
@@ -19,6 +19,13 @@ def render(markdown):
     ))
 
 
+def rtlo_scrub(markdown):
```

Comment:
Can't this be done within `render_and_scrub`? Is there a case where we would want to use `render_and_scrub` without this functionality?